Tasks/AzureStorage/operations/AzureBlobUtils.ts

```diff
@@ -0,0 +1,32 @@
+import tl = require("vsts-task-lib/task");
+import msRestAzure = require("azure-arm-rest/azure-arm-common");
+import armStorage = require("azure-arm-rest/azure-arm-storage");
+import azcopy = require("./AzCopyUtils");
+import util = require("util");
+
+export class AzureBlobUtils {
+    public static armStorageClient: armStorage.StorageManagementClient;
+    public static azCopyExeLocation: string;
+
+    constructor(client: armStorage.StorageManagementClient, azCopyExeLocation: string) {
+        AzureBlobUtils.armStorageClient = client;
+        AzureBlobUtils.azCopyExeLocation = azCopyExeLocation;
+    }
+
+    public downloadFromBlob(storageAccountName: string, containerName: string, commonVirtualPath:string, destLocation: string): Q.Promise<void> {
```

Comment:
This function will download all blobs inside container folder. Rename to something like "downloadBlobs"